rpcs3/Emu/RSX/VK/VKOverlays.cpp

```diff
@@ -899,27 +899,60 @@ namespace vk
 			"	int limit_range;\n"
 			"	int stereo;\n"
 			"	int stereo_image_count;\n"
+			"	int stereo_display_method;\n"
 			"};\n"
 			"\n"
 			"vec4 read_source()\n"
 			"{\n"
 			"	if (stereo == 0) return texture(fs0, tc0);\n"
 			"\n"
-			"	vec4 left, right;\n"
+			"	vec4 left, right, final;\n"
 			"	if (stereo_image_count == 2)\n"
 			"	{\n"
-			"		left = texture(fs0, tc0);\n"
-			"		right = texture(fs1, tc0);\n"
+			"   vec2 coord_left = tc0 * vec2(1.f, 0.4898f);\n"
```

Comment:
The formatting in your added code is completely off.
The indentations are all over the place.
You're missing spaces in "if(" and after "//" in comments